.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        node-version: [12.x, 14.x, 16.x, 17.x]
+        node-version: [12.20.0, 12.x, 14.x, 16.x, 17.x]
```

Comment:
Is it worth testing both 12.20.0 and latest 12.x, or would it be enough with just 12.20.0?

Reply:
Felt like both the explicit minimum version, and the likely version are worth checking but :shrug: